src/FSharp.AWS.DynamoDB/TableContext.fs

```diff
@@ -285,20 +275,21 @@ type TableContext<'TRecord> internal
         if template.PrimaryKey <> rd.PrimaryKey then
             invalidArg (string typeof<'TRecord2>) "incompatible key schema."
 
-        new TableContext<'TRecord2>(client, tableName, rd, defaultMetricsCollector)
+        new TableContext<'TRecord2>(client, tableName, rd, metricsCollector)
 
+    /// Creates a new table context instance that uses a unique metricsCollector
```

Comment:
"Unique" does not convey much
How about
`/// Creates an identical table context with the specified metricsCollector callback replacing any previously specified one`